spec.md

```diff
@@ -499,6 +499,14 @@ message PluginCapability {
       // attempt to invoke the REQUIRED ControllerService RPCs, as well
       // as specific RPCs as indicated by ControllerGetCapabilities.
       CONTROLLER_SERVICE = 1;
+
+      // ACCESSIBILITY_CONSTRAINTS indicates that the volumes for this
+      // plugin may not be equally accessible by all nodes in the
+      // cluster. The CO MUST use the topology information returned by
+      // CreateVolumeRequest along with the topology information
+      // returned by NodeGetId to ensure that a given volume is
+      // accessible from a given node when scheduling workloads.
+      ACCESSIBILITY_CONSTRAINTS = 2;
```

Comment:
I am curious why this capability is needed. Can the CO just infer that based on the result of `GetNodeInfo`?

Reply:
If the CO wants to be able to quickly handle plugins with `ACCESSIBILITY_CONSTRAINTS` differently from other plugins. I don't have a concrete use case in mind though, so I'm willing to drop this.

Reply:
I believe that we discussed adding this capability so that the CO can ignore or reject plugins that require topology until the CO can honor topology requirements.  For CloudFoundry, we will likely pick up v1.3 before we have topology implemented so this capability would be useful for us.